spring-cloud-task-jdbchdfs-common/src/main/java/org/springframework/cloud/task/jdbchdfs/common/JdbcHdfsTaskProperties.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.task.jdbchdfs.common;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Establishes the Configuration Properties for the JdbcHdfs task.
+ * @author Glenn Renfro
+ */
+@ConfigurationProperties("jdbchdfs")
+public class JdbcHdfsTaskProperties {
+
+	public static final String DEFAULT_FS_URI = "hdfs://localhost:8020";
+
+	public static final int DEFAULT_PARTITION_COUNT = 4;
+
+	public static final String DEFAULT_FILE_NAME = "jdbchdfs";
+
+	public static final long DEFAULT_ROLLOVER = 1000000000;
+
+	public static final String DEFAULT_DIRECTORY = "/data";
+
+	public static final String DEFAULT_FILE_EXTENSION = "csv";
+
+	public static final String DEFAULT_DELIMITER = ",";
+
+	public static final int DEFAULT_COMMIT_INTERVAL = 1000;
+
+	public static final int DEFAULT_MAX_WORKERS = 2;
+
+	/**
+	 * The URI to the hadoop file system.
+	 */
+	private String fsUri = DEFAULT_FS_URI;
```

Comment:
could we let this default to `null` so that it matches the way Hadoop configuration defaults. That way we could override it with either `spring.hadoop.fs-uri` or `jdbchdfs.fs-uri`. The way it is now the DEFAULT_FS_URI overrides any `spring.hadop.fs-uri` settings, which to me is unfortunate. If we do this we can remove the DEFAULT_FS_URI constant. This seems like a random default anyway - Hadoop itself uses `file:///` as the default.